test/classes/ferguson/inherit-generic-q.good

```diff
@@ -0,0 +1,2 @@
+inherit-generic-q.chpl:4: warning: inherits expr should include (?) for a generic superclass
+inherit-generic-q.chpl:8: error: inherits expr should not include (?) for a concrete class
```

Comment:
I see the term is used here too, which may make it slightly harder to change in just the error above.  It feels unambiguous to me when it leads off the sentence as in this case.